core/drivers/counter/stm32_stgen.c

```diff
@@ -155,7 +155,7 @@ static void stm32_stgen_pm_resume(void)
 	io_setbits32(stgen_d.base + STGENC_CNTCR, STGENC_CNTCR_EN);
 
 	DMSG("Time spent in low-power: %"PRIu64"ms",
-	     (nb_pm_count_ticks * 1000) / clock_src_rate);
+	     (uint64_t)(nb_pm_count_ticks * 1000) / clock_src_rate);
```

Comment:
`nb_pm_count_ticks` is of type `signed long long` so the `DMSG()` format specifier should be `lld` instead of `"PRIu64"`.

Reply:
Oops, I didn't see Jérôme's comment :-)